ui_release.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+set -e
+
+cd ui/
+
+files=("../LICENSE" "../CHANGELOG.md" ".npmignore")
```

Comment:
mmm I would tend to avoid a duplication of the `LICENSE` file. Even if we won't change the license, that would be weird to have it duplicated everywhere.
Yeah I have no idea about the `CHANGELOG.md`. I'm feeling after a 1 or 2 releases, we will know what do to :)